README.md

```diff
@@ -81,12 +81,12 @@ cd iceberg-docs
 
 To start the landing page site locally, run:
 ```shell
-cd landing-page && hugo serve
+(cd landing-page && hugo serve)
 ```
 
 To start the documentation site locally, run:
 ```shell
-cd docs && hugo serve
+(cd docs && hugo serve)
```

Comment:
It's so that you can run it from the root while remaining in the same directory. That way you can alternate between running the docs site and the landing page without having to remember to cd back to root. If this makes the command less intuitive I'm ok with taking it out. What do you think?